console/frontend/src/main/frontend/src/app/views/status/status.component.html

```diff
@@ -1,66 +1,82 @@
 <div class="wrapper wrapper-content">
   <!--     <uib-alert ng-repeat="alert in alerts | orderBy:'type'" type="{{alert.type}}" close="closeAlert($index)">{{alert.message}}</uib-alert> -->
 
-  <ul *ngIf="alerts.length > 0" class="alert p-n m-n">
+  <ul class="alert p-n m-n">
     <li
-      *ngIf="messageLog[selectedConfiguration].errorStoreCount === 1"
+      *ngIf="
+        freeDiskSpacePercentage &&
+        serverInfo &&
+        freeDiskSpacePercentage < FREE_DISK_SPACE_ALERT_THRESHOLD
+      "
       class="list-group-item alert-danger"
     >
-      <i class="m-r-md fa fa-info-circle"></i>
-      <span
-        >Errorlog contains 1 record. Service management should check whether
-        this record has to be resent or deleted</span
-      >
+      Disk space is running low! Only
+      {{ serverInfo.fileSystem.freeSpace | humanFileSize }} available, that's
+      {{ round(freeDiskSpacePercentage) }}% of the total disk size ({{
+        serverInfo.fileSystem.totalSpace | humanFileSize
+      }})
     </li>
-    <li
-      *ngIf="messageLog[selectedConfiguration].errorStoreCount > 1"
-      class="list-group-item alert-danger"
-    >
-      <i class="m-r-md fa fa-info-circle"></i>
-      <span
-        >Errorlog contains
-        {{ messageLog[selectedConfiguration].errorStoreCount }} records. Service
-        management should check whether this record has to be resent or
-        deleted</span
+    <ng-template *ngIf="alerts.length > 0">
```

Comment:
This *ngIf wont properly work on ng-template, see https://stackoverflow.com/a/44838187